node/service/Cargo.toml

```diff
@@ -96,23 +95,21 @@ pallet-schemas-runtime-api = { path = "../../pallets/schemas/src/runtime-api" }
 substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.29" }
 
 [features]
-default = ["std"]
+default = ["std", "frequency"]
```

Comment:
@aramikm  cargo check will work now, since I am used to make file, adding a default runtime was needed